src/App.js

```diff
@@ -1,27 +1,37 @@
-import logo from './logo.svg';
-import './App.css';
+import Screen from './components/screen';
+import ButtonsContainer from './components/buttons-container';
```

Comment:
- [ ] Please when you will create your `Calculator.js` component please make sure that you import it here in your `App.js` and also make sure that your calculator.js match the proposed design. :+1:

Reply:
Done!